### DISCUSSION TITLE
Error in osg202-osg.dll

### DISCUSSION THREAD

**Discussion by AhmedAredah** [openscenegraph/OpenSceneGraph#1311] [General]
C:\Program Files\OpenSceneGraph\bin\osg202-osg.dll : fatal error LNK1107: invalid or corrupt file: cannot read at 0x300

I am getting this error when building a project that depends on osg.dll. Any idea why I am getting this error?

**Comment by robertosfield** [comment]
The clue is in the error message, somewhere along the line the .dll you have chosen to use is corrupted, this can happen for many reasons but as this project only provides the source could for developers to build into their own dll's supporting you and your acquired/built dll's is not something we can do.
